crates/graphql/src/schema.rs

```diff
@@ -181,6 +181,50 @@ impl<'a> From<models::MetadataCreator<'a>> for NftCreator {
     }
 }
 
+#[derive(Debug, Clone)]
+struct NftAttribute {
+    metadata_address: String,
+    value: String,
+    trait_type: String,
+}
+
```

Comment:
Should this be adorned with a juniper attribute?

Reply:
Yep!